index.html

```diff
@@ -1,697 +1,715 @@
+        width: 100%;
+        text-align: left;
+    }
+    .get-a-quote {
+        background: #f4f8f9;
+        border: 1px solid #f4f8f9;
+        box-shadow: 0 0 4px rgba(0,0,0,0.2), 0 3px 6px rgba(0,0,0,0.2);
+        padding: 0;
+        margin-bottom: 100px;
+        color: #070d14;
+        width: 100%;
+        max-width: 500px;
+    }
+    .get-a-quote a {
+        color: #17395b;
+    }
+    .get-a-quote .header {
+        color: white;
+        padding: 15px;
+        background: #2c3e50;
+    }
+    .get-a-quote .tabs {
+        display: flex;
+        background: #2c3e50;
+        border-left: 8px solid #2c3e50;
+    }
+    .get-a-quote .tabs .tab {
+        background: #c3cacc;
+        color: #4d637a;
+        font-size: 0.7em;
+        text-transform: uppercase;
+        padding: 7px 15px;
+        border-right: 1px solid #2c3e50;
+        border-bottom: 1px solid #2c3e50;
+        cursor: pointer;
+    }
+    .get-a-quote .tabs .tab.active {
+        background: #f4f8f9;
+        color: inherit;
+        border-bottom: 1px solid transparent;
+    }
+    .get-a-quote .tab-content {
+        display: none;
+    }
+    .get-a-quote .tab-content > * {
+        margin: 15px;
+    }
+    .get-a-quote .tab-content.active {
+        display: block;
+    }
+    .get-a-quote .tab-content label {
+        text-align: center;
+        display: block;
+    }
+    .get-a-quote .tab-content input {
+        text-align: right;
+        font-family: inherit;
+        font-weight: 200;
+        padding: 0.2em;
+        font-size: 1.2em;
+        width: 6em;
+    }
+    .get-a-quote .evaluation-text {
+        background: #cee8ef;
+        margin-left: 0;
+        margin-right: 0;
+        padding: 5px 15px;
+        font-size: 0.7em;
+        min-height: 4em;
+        transition: background 1.2s;
+    }
+    .get-a-quote .evaluation-text.good {
+        background: #c8efba;
+    }
+    .get-a-quote .evaluation-text.bad {
+        background: #efe6ba;
+    }
+    .get-a-quote .tab-content .button {
+        display: block;
+        width: 100%;
+        text-align: center;
+        background: #2c3e50;
+        border: none;
+        cursor: pointer;
+        text-decoration: none;
+        padding: 0.5em;
+        font-size: 1.4em;
+        color: white;
+        font-weight: 200;
+        font-family: inherit;
+        margin-left: 0;
+        margin-right: 0;
+        margin-bottom: 0;
+        transition: background-color 0.2s;
+    }
+    .get-a-quote .tab-content .button:hover {
+        background: #3f5e7c;
+    }
+    .get-a-quote .get-in-touch-explanation {
+        text-align: center;
+        font-size: 0.6em;
+    }
+</style>
+<section class="rest-sales">
+    <h1>Get a quote</h1>
+    <p>
+        We prefer working on a monthly budget.
+        We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
+    </p>
```

Comment:
Maybe a note about have flexibility as a key value, so lets find a solution that suits _your_ business.